### PR TITLE
Support GUI of dumpParser

### PR DESCRIPTION
**Pull request by junmin-kim**
Using tkinter, this python script set arguments of dumpPaser
and redirect them to dumpParser.py

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@junmin-kim Could you modify how to use ramdump doc? https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseRamdump.md

**Comment by sunghan-chang** [comment]
@junmin-kim And later, we should provide a functionality to see result in this GUI.

**Comment by junmin-kim** [comment]
Please don't merge. I'm fixing it

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToUseRamdump.md [hunk 1]
```diff
@@ -54,10 +54,16 @@ copying ramdump_0x02020000_0x0210c800.bin to  $TIZENRT_BASEDIR/build/output/bin
 
 ## Ramdump Parsing Steps
 1. Run Ramdump Parser Script
+1-1. CUI
 ```
 cd $TIZENRT_BASEDIR/tools/ramdump/
 python dumpParce.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
+1-2. GUI
```
Comment (sunghan-chang) [inline docs/HowToUseRamdump.md]: Please write more details.
1. What is different between them.
2. User can execute one among them, not all of them.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**